File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
while 1:
    a, b, c = map(int, input().split())

    if a == b == c == 0:
        break
    
    numbers = sorted([a, b, c])
    
    if numbers[2] ** 2 == numbers[0] ** 2 + numbers[1] ** 2:
        print('right')
    else:
        print('wrong')

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.java
import java.util.Scanner;
import java.util.Arrays;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        while (true) {
            int[] tri = new int[3];
            
            tri[0] = sc.nextInt();
            tri[1] = sc.nextInt();
            tri[2] = sc.nextInt();
            
            if (tri[0] == 0 && tri[1] == 0 && tri[2] == 0) break;
            
            Arrays.sort(tri);
            
            if (tri[0] * tri[0] + tri[1] * tri[1] == tri[2] * tri[2]) System.out.println("right");
            else System.out.println("wrong");
        }
    }
}